src/nvim/lua/executor.h

```diff
@@ -35,6 +35,16 @@ EXTERN int nlua_refcount INIT(= 0);
     } \
   } while (0)
 
+// Set lua sctx for verbose output when a lua file is being sourced
+#define NLUA_SET_SCTX(x) \
```

Comment:
this could just be a function.

Reply:
You're correct . I'll convert it unto a function.